models/blobs/ethereum/transactions.sql

```diff
@@ -0,0 +1,92 @@
+{{ config(
+    schema = 'blobs_ethereum',
+    alias = 'blobs_transactions',    
```

Comment:
Also, should be call this table something like `blobs_enriched` since this one isn't at the "transaction" level?